local-cli/generator-ios/templates/xcodeproj/project.pbxproj

```diff
@@ -479,6 +510,20 @@
 			remoteRef = 146834031AC3E56700842450 /* PBXContainerItemProxy */;
 			sourceTree = BUILT_PRODUCTS_DIR;
 		};
+        5E9157331DD0AC6500FF2AA8 /* libRCTAnimation.a */ = {
+            isa = PBXReferenceProxy;
+            fileType = archive.ar;
+            path = libRCTAnimation.a;
+            remoteRef = 5E9157321DD0AC6500FF2AA8 /* PBXContainerItemProxy */;
+            sourceTree = BUILT_PRODUCTS_DIR;
+        };
+        5E9157351DD0AC6500FF2AA8 /* libRCTAnimation-tvOS.a */ = {
```

Comment:
I haven't done any Apple TV development yet, and I am not familiar with how Xcode handles static libraries for projects that may target tvOS. @dlowder-salesforce perhaps you can chime in?